devops/actions/aws-ec2/aws-ec2.js

```diff
@@ -129,7 +138,13 @@ async function start(param_type, param_label, param_ami, param_spot, param_disk,
 async function stop(param_label) {
   // last error that will be thrown in case something will break here
   let last_error;
-  const ec2 = new AWS.EC2();
+  const ec2 = new EC2({
```

Comment:
Please try to avoid this code duplication with start function and pass ec2 as start/stop function param.